lib/components/artwork_grids/infinite_scroll_grid.js

```diff
@@ -88,64 +61,45 @@ class InfiniteScrollArtworksGrid extends React.Component {
 
   constructor(props) {
     super(props)
+
     this.state = {
       sectionDimension: 0,
-      artworks: this.props.artworks,
-      page: this.props.artworks.length ? 1 : 0,
       completed: false,
       fetchingNextPage: false,
     }
 
     this._sentEndForContentLength = null
-  }
-
-  // Initial setup
-  componentWillMount(){
-    if (this.state.artworks.length === 0) {
-      this.fetchNextPage()
+    if (this.props.filter) {
```

Comment:
Btw, I’m thinking, if you’d move this down to https://github.com/artsy/emission/pull/441/files/f2e35653d3af57d27663c4a93b34e89a71ec3b9e..3d7c0d0693b1d9fc4f590c95d43e4f37e5adae14#diff-4ae33428e5de3c1de5112e081353e759R105, then maybe it’s a bit nicer as at least the setting of variables is done in one place?